corpus/templates/pages/sig.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}
+{{sig.name}}
+{% endblock %}
+
+{% block content %}
+<!-- Hero Section -->
+<!-- TODO: Add action button -->
+<section
```

Comment:
Can you make the section only be 50% of the total screen height? Check the implementation on the [About Us page](https://github.com/IEEE-NITK/corpus/blob/d82df72986409f425e41bbb522433d2f0d135bb8/corpus/templates/pages/about_us.html#L14-L20)

Reply:
done